aform/src/components/form/AFileAttach.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <button type="button" @click="open()">
+    {{ props.label }}
+  </button> &nbsp;
+  <button type="button" :disabled="!files" @click="reset()">
+    Reset
+  </button>
+
+  <template v-if="files">
+    <div>
+      <p>You have selected: <b>{{ `${files.length} ${files.length === 1 ? 'file' : 'files'}` }}</b></p>
+      <li v-for="file of files" :key="file.name">
+        {{ file.name }}
+      </li>
+    </div>
+  </template>
+</template>
+
+<script setup lang="ts">
+import { useFileDialog } from '@vueuse/core'
+import { SchemaTypes } from 'types/index';
+
+const props = defineProps<{
+  schema: SchemaTypes[]
+  label: string
+  collapsible?: boolean
+  data?: any
+}>()
+
+const { files, open, reset } = useFileDialog()
```

Comment:
@bhattdevarsh Question about using this in a client/server context: how do we call an upload API? Should we provide

Reply:
Got it. This is related to resolving #120 in that case

Reply:
@bhattdevarsh this one can move ahead with v10.9.0